systems/controllers/osc/osc_tracking_data.cc

```diff
@@ -25,24 +25,26 @@ using multibody::makeNameToPositionsMap;
 using multibody::makeNameToVelocitiesMap;
 
 /**** OscTrackingData ****/
-OscTrackingData::OscTrackingData(
-    const string& name, int n_r, const MatrixXd& K_p, const MatrixXd& K_d,
-    const MatrixXd& W, const MultibodyPlant<double>* plant_w_spr,
-    const MultibodyPlant<double>* plant_wo_spr)
+OscTrackingData::OscTrackingData(const string& name, int n_y, int n_ydot,
+                                 const MatrixXd& K_p, const MatrixXd& K_d,
+                                 const MatrixXd& W,
+                                 const MultibodyPlant<double>& plant_w_spr,
```

Comment:
I think using const reference in our code doesn't make any difference. I changed it because of the code style.Drake style guide says "Input parameters are usually values or const references, while output and input/output parameters will be pointers to non-const."